Gemfile

```diff
@@ -12,6 +12,10 @@ group :development, :test do
   gem 'pg_jruby', require: nil, platform: :jruby
   gem 'pond', require: nil
   gem 'rubocop'
+  gem 'rubocop-performance'
```

Comment:
Part of the issue with this repo always having linting errors is that these aren't pinned in any way so there will never be a PR to bump them where we could preemptively fix the issues.

Suggest pinning these to a specific version

Reply:
🧷